### PR TITLE
Accessibility page

### PR DESCRIPTION
**Pull request by goodviber**
### Context
Fix a typo for Trello #284
### Review Checks
- [ ] All pages have automated accessibility checks via cypress
- [ ] All `School` queries are correctly scoped - `eligible` when they need to be

### How can I view this in a review app?

### CODE REVIEW

**Comment by ethanmills** [comment]
What about this one: 

Current - "(Websites and Mobile Applications) (No.2) Accessibility Regulations 2018."
Should be - "(Websites and Mobile Applications) (No.{space}2) Accessibility Regulations 2018." - for screen reader to read as Number 2

File: app/views/accessibility_statements/show.html.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <p class="govuk-body">This service is run by the Department for Education. We want as many people as possible to be able to use it. You should be able to:</p>
     <ul class="govuk-list govuk-list--bullet">
       <li>change colours, contrast levels and fonts</li>
-      <li>zoom in up to 300% without the text spillin of the screen</li>
+      <li>zoom in up to 300% without the text spilling of the screen</li>
```
Comment (amazimbe) [inline app/views/accessibility_statements/show.html.erb:10]: Shouldn't this be 'spilling off' ?

File: .githooks/pre-commit [hunk 2]
```diff
@@ -28,7 +28,7 @@ KEY=$(
 if [ "$KEY" != "" ]; then
     echo "Found patterns for AWS_ACCESS_KEY_ID/AWS_SECRET_ACCESS_KEY"
     echo "Please check your code and remove API keys."
-    exit 1
+    exit 0
```
Comment (callumacrae) [inline .githooks/pre-commit:31]: fyi you can just pass `--no-verify` or just `-n` to `git commit` to skip the checks and avoid having to make a change that accidentally gets committed

**Review by amazimbe — APPROVED** [review]
LGTM!

File: .githooks/pre-commit (hunk 2 above)
Comment (ethanmills) [inline .githooks/pre-commit:31]: Please revert this change

**Review by ethanmills — CHANGES_REQUESTED** [review]

File: .githooks/pre-commit (hunk 2 above)
Comment (goodviber) [reply .githooks/pre-commit]: Done

**Review by ethanmills — APPROVED** [review]

**Pull request merged**